scripts/analysisTools/tests/buildPseudodata.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
```

Comment:
Out of curiosity what behaviour do you get here that you don't have from the usual --pseudodata option ?